lib/sport_ngin_aws_auditor/scripts/audit.rb

```
@@ -17,13 +17,22 @@ class << self
       #################### EXECUTION ####################
 
       def self.execute(environment, options, global_options)
-        aws(environment, global_options)
-        collect_options(environment, options, global_options)
-        print_title
-        @regions.each { |region| audit_region(region) }
-        reset_credentials
+        begin
+          aws(environment, global_options)
+          collect_options(environment, options, global_options)
+          print_title
+          @regions.each { |region| audit_region(region) }
+          reset_credentials
+        rescue Exception
+          if options[:slack]
+            NotifySlack.new("Sorry, something seems to have gone wrong.", options[:config_json]).perform
+          else
```

Hmm let's raise this error anyway just post to slack first so something like

```ruby
rescue StandardError => e
  NotifySlack.new("Sorry, something seems to have gone wrong.", options[:config_json]).perform if options[:slack]
  raise e
```